tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -109,7 +109,10 @@ const char* LayerTypeToString(nvinfer1::LayerType layer_type) {
     ADD_LAYER(CONCATENATION)
     ADD_LAYER(ELEMENTWISE)
     ADD_LAYER(PLUGIN)
+#if !IS_TRT_VERSION_GE(8, 0, 0, 0)
+    // The TRT IRNNv2Layer has been deprecated in favor of the loop API
     ADD_LAYER(RNN)
+#endif
     ADD_LAYER(UNARY)
     ADD_LAYER(PADDING)
     ADD_LAYER(SHUFFLE)
```

Comment:
Need to add this to the end of the list:
#if IS_TRT_VERSION_GE(8, 0, 0, 0)
    ADD_LAYER(QUANTIZE)
    ADD_LAYER(DEQUANTIZE)
#endif

Reply:
I see code change for this. In the future, please response to the comment so I know you already addressed it.

Reply:
Added the quantize conversions and moved the TRT8 check to the bottom to keep things slightly cleaner, thanks